pkg/resourcecreator/ingress/ingress.go

```diff
@@ -155,7 +173,11 @@ func nginxIngresses(source Source, cfg Config) ([]*networkingv1.Ingress, error)
 		ingressClass := util.ResolveIngressClass(rule.Host, cfg.GetGatewayMappings())
 
 		if ingressClass == nil {
-			return nil, fmt.Errorf("domain '%s' is not supported", rule.Host)
+			return nil,
+				fmt.Errorf("domain '%s' is not supported in this cluster. You are attempting to add your ingress to '.%s', try one of these '%v'",
```

Comment:
Something about the wording here that is a bit off. It matters which one of the domains you choose, as this directly affects which audience the application is exposed to. I think we should refer to the documentation if we start suggesting domains

Reply:
> I think we should refer to the documentation if we start suggesting domains

https://www.w3.org/WAI/WCAG21/Understanding/error-suggestion.html

^ The helpful thing to do when we know what is wrong is to suggest how to fix it. I think this is a good principle for error messages. 

I want to have immediately actionable errors for this. Previously, this was the error: 
`[2024-08-22T11:13:10.617657017Z] Status: in_progress: Application/nks-vdb (FailedGenerate): domain 'nksvdb.ansatt.nav.no' is not supported`. Everybody knows that `$FOO.ansatt.nav.no` is supported. 

Now the error is 
`domain 'nksvdb.ansatt.nav.no' is not supported in this cluster. You are attempting to add your ingress to '.ansatt.nav.no', try one of these '[.ansatt.dev.nav.no, .intern.dev.nav.no, .dev-gcp.nais.io, .very-intern....]'`

which has 
1. the original error with the input 
2. How Naiserator considered the input 
3. a list of suggestions for remediating the error. 

Sending people to rtfm is, imho, the wrong approach here since we know what's wrong and can suggest how to fix it.

Reply:
I completely agree, my point was the wording. It sounds as if you should just "try one of these out", and I mean that without referring to any documentation that is a bad idea since it can have serious consequences if they try the wrong one.

Reply:
@Reasonable-Solutions fixed it (added docs link) in a9d37f226

Reply:
Hvorfor har brukerne våre mulighet til å bruke "feil" ingress?